config/assisted-service/kustomization.yaml

```diff
@@ -1,8 +1,4 @@
 resources:
-# TODO: remove scality after local volume support has been added
-- scality.yaml
-- scality-secret.yaml
```

Comment:
By removing scality-secret.yaml and scality-public-secret.yaml I think we left ourselves with a complicated situation. The assisted-service deployment spec still references them. assisted-service pod cannot successfully initialize because the secrets are missing.

I think we have two choices, the first is to actually leave scality-secret.yaml and scality-public-secret.yaml in and just remove scality.yaml (the deployment). 

The second, and perhaps cleaner approach, would be to remove the references to these secrets in the operator's assisted-service deployment spec. Example: https://github.com/openshift/assisted-service/blob/master/deploy/assisted-service.yaml#L81

Reply:
I'm not sure we can remove that secret quite yet because of how that might effect the deployment that assisted-test-infra uses. I'll look into that today.